system/jlib/jlog.cpp

```diff
@@ -2295,7 +2295,14 @@ MODULE_INIT(INIT_PRIORITY_JLOG)
     thePassAllFilter = new PassAllLogMsgFilter();
     thePassLocalFilter = new PassLocalLogMsgFilter();
     thePassNoneFilter = new PassNoneLogMsgFilter();
-    theStderrHandler = new HandleLogMsgHandlerTable(stderr, MSGFIELD_STANDARD);
+    unsigned  msgFields = MSGFIELD_STANDARD;
+#ifdef _CONTAINERIZED
+    const char *logFields = queryEnvironmentConf().queryProp("logfields");
+    if (!isEmptyString(logFields))
+        msgFields = LogMsgFieldsFromAbbrevs(logFields);
+#endif
```

Comment:
we shouldn't be environment.conf in _CONTAINERIZED mode, or rather we shouldn't be adding anything new to it.
So prob. don't want this change.
logging/@logFields will be a property picked up from the config queryComponentConfig() when the container log handler is installed.